cypress/e2e/email-address.cy.js

```diff
@@ -0,0 +1,67 @@
+describe("email address", () => {
+  before(() => {
+    cy.visit("/email-address");
+  });
+
+  context("back link", () => {
+    beforeEach(() => {
+      cy.get('[data-testid="back-link"]').as("button");
+    });
+
+    it("displays", () => {
+      cy.get("@button").should("be.visible");
+    });
+
+    it("says 'Back'", () => {
+      cy.contains("Back");
+    });
+
+    it("links to the right page", () => {
+      cy.get("@button").click();
+      cy.url().should("include", "date-of-birth");
+    });
+  });
+
+  context("email field", () => {
+    beforeEach(() => {
+      cy.get('[data-testid="email-field"]').as("field");
+      cy.get('[data-testid="email-field"]').as("label");
+    });
+
+    it("displays", () => {
+      cy.get("@field").should("be.visible");
+      cy.get("@label").should("be.visible");
+    });
+
+    it("has a label saying 'What is your email address?'", () => {
+      cy.get("@label").contains("What is your email address?");
```

Comment:
In the HTML you haven't added the test label to this element

Reply:
I've added the test label and that probably fixed an additional issue that would cause tests to fail if the initial failing test was fixed.

This is a failing test that is keep failing:

```
     CypressError: `cy.visit()` failed trying to load:

/email-address

We failed looking for this file at the path:

/Users/miroslav.palka/code/Porim/project-ananas/email-address

The internal Cypress web server responded with:

  > 404: Not Found
```